docs/setup/independent/install-kubeadm.md

```diff
@@ -51,6 +51,24 @@ The pod network plugin you use (see below) may also require certain ports to be
 open. Since this differs with each pod network plugin, please see the
 documentation for the plugins about what port(s) those need.
 
+## Installing ebtables ethtool
+
+If you see warning while running ```kubeadm init```
```

Comment:
suggested edit: "If you see the following warnings while running \`kubeadm init\`"